content/en/events/2023/kcscn/schedule.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Schedule
+type: docs
+weight: 3
+---
+
+**NOTE:** You **MUST** pick up your badge before attending the summit event.
+
+## Steering AMA
+
+The Summit will start off with an AMA with members of the Kubernetes Steering
+Committee.  Bring any and all questions about how we're running the project,
+ongoing efforts & issues, and whatever you'd like to discuss with Steering.
+
+## Unconference
+
+The Contributor Summit will be primarily an [Unconference-style event](https://blog.crisp.se/2016/08/30/henrikkniberg/what-is-an-unconference).  This means that
+you should arrive with topics in mind, and we will pitch them and vote on them
+in the morning, and then hold sessions in the afternoon, after the Steering AMA.
+This is the format used at many [DevOps Days](https://devopsdays.org/open-space-format/), and helps us accommodate changing 
+needs and unpredictable travel schedules.
+
+If you have a topic in mind, please add it to the [GitHub Issue](https://github.com/kubernetes/community/issues/7157).
+
+## Doc Sprint
+
+SIG-Docs will have a full-day Documentation working session during the Summit.
+If you contribute to the docs -- and who doesn't -- or you want to learn to
+do it better, join them.
+
```

Comment:
```suggestion
```

we don't have docs sprint this time, right?

Reply:
It will depend on the Registration/CFP survey. If some contributors want it and have it in the CFP, we may choose it as a short session as we have several sig-doc-zh owners.